packages/shared/src/components/search/SearchBar.spec.tsx

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Test have lots of comments/commented out code, it's not really neat to merge until they are cleaned/removed rather.
If you can't resolve them, add them to a ticket rather.